core/src/main/java/org/geysermc/geyser/entity/type/player/SessionPlayerEntity.java

```diff
@@ -122,6 +119,16 @@ public void setFlags(ByteEntityMetadata entityMetadata) {
         refreshSpeed = true;
     }
 
+    @Override
+    public void setBoundingBoxHeight(float height) {
```

Comment:
You could change this to a boolean return type so you can just check `if (super.setBoundingBoxHeight(height))`.